services/galley/src/Galley/API/Action.hs

```diff
@@ -36,6 +36,8 @@ module Galley.API.Action
   )
 where
 
+-- import qualified Brig.Types.User as User
+
```

Comment:
```suggestion
```

I tend to:
1. Commit only chunks I've looked at again individually (there are lots of good tools for that besides emacs's magit)
2. Remove imports I don't need rather than commenting them out, and if I change my mind get them back by undoing the change in git.

git is nice!  :)